README.md

```diff
@@ -1,7 +1,5 @@
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/alvarcarto/url-to-pdf-api)
 
-[![Build Status](https://travis-ci.org/alvarcarto/url-to-pdf-api.svg?branch=master)](https://travis-ci.org/alvarcarto/url-to-pdf-api)
-
```

Comment:
This can be replaced like this if you want to: https://github.com/kesko-dev/kesko-push/pull/41/commits/6a340294f65d94cddec7f5ae24189420eff6c387

Reply:
Thanks, it's replaced now!